Swift-Base/Constants.swift

```diff
@@ -12,6 +12,10 @@ import UIKit
 let kAPIKeyCrashlitycs  = ""
 let kAPIKeyGoogle       = ""
 
+/*--------------User Defaults keys-------------*/
+let kUserDefaultsDeviceTokenData = "kUserDefaultsDeviceTokenData"
+let kUserDefaultsDeviceTokenString = "kUserDefaultsDeviceTokenString"
+
 /*----------Notifications---------*/
 let kMockNotification:String = "mock_notification";
```

Comment:
@NikolaevSergey @VladimirGoncharov can we use names without ```k``` prefix for notification names? Maybe we should replace ```kMockNotification``` with ```MockNotification```. I use ```k``` prefix only for keys in KVC and all kind of dictionaries.